common/lib/xmodule/xmodule/lti_2_util.py

```diff
@@ -231,9 +231,9 @@ def clear_user_module_score(self, user):
         Returns:
             nothing
         """
-        self.set_user_module_score(user, None, None)
+        self.set_user_module_score(user, None, None, score_deleted=True)
```

Comment:
Are this and the downstream associated changes a bug fix separate from the removal of `WRITE_ONLY_IF_ENGAGED`?

Reply:
Yes, it's a bug fix that was hidden - since all of our tests didn't run with `WRITE_ONLY_IF_ENGAGED` enabled.
I have a subsequent PR that enables all grades waffle flags that are currently enabled in prod, so we're testing what's in production.  The only such remaining flag, after this PR, is assume_zero_grade_if_absent.